server/src/main/java/org/spine3/server/event/EventBus.java

```diff
@@ -191,7 +191,7 @@ public void post(Event event) {
     @Override
     protected Event enrich(Event event) {
         if (enricher == null ||
-                !enricher.canBeEnriched(event)) {
+            !enricher.canBeEnriched(event)) {
             return event;
```

Comment:
Something wrong with `Idea`'s settings, fix it please. An if statement condition and a body have same indent. Such formatting is wrong.